scanner/cmd/updater/main.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 func tryExport(outputDir string) error {
-	const timeout = 1 * time.Hour
+	const timeout = 2 * time.Hour
```

Comment:
Maybe we want to make this a command line parameter, so we can pass an argument, and update this in the future without having to change the  updater logic.

Reply:
I'll keep it as-is for this PR, as we do plan on backporting the vuln updating process improvements you and Yi are working on for 4.4 and 4.5